requirements.txt

```diff
@@ -2,6 +2,7 @@ django >= 3.2, <3.3
 xlrd == 2.0.1
```

Comment:
Do we still use this somewhere?

Reply:
Indeed, we use to test exporters.

Reply:
then this should move to `requirements-dev`